config/environments/production.rb

```diff
@@ -15,52 +17,52 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
```

Comment:
What does this change do exactly?

Reply:
This came in with the `Adds all new/adjusted files.` commit that was focused on matching all of the code changes that the newer Blacklight version implemented when changing Rails versions. In this case, I made sure that all of the matched code wasn't altered by us beforehand.